src/components/related-posts/related-posts-card.astro

```diff
@@ -1,20 +1,32 @@
 ---
 import style from "./related-posts-card.module.scss";
+import { getHrefContainerProps } from "utils/href-container-script";
+import { UnicornInfo } from "types/UnicornInfo";
 
 interface RelatedPostsCardProps {
 	title: string;
 	slug: string;
-	authors: string[];
+	authors: UnicornInfo[];
 }
 
 const { title, slug, authors } = Astro.props as RelatedPostsCardProps;
 ---
 
-<a class={style.container} href={`/posts/${slug}`}>
+<span
+	{...getHrefContainerProps(`/posts/${slug}`)}
+	class={style.container}
+	href={`/posts/${slug}`}
+>
 	<span class={`text-style-body-medium-bold ${style.articleTitle}`}
 		>{title}</span
 	>
```

Comment:
```suggestion
>
	<a
		href={`/posts/${slug}`}
		class={`text-style-body-medium-bold ${style.articleTitle}`}
	>
		{title}
	</a>
```
The "tsc" check is erroring on the `<span>` here - `href` needs to be moved to an `<a>` element. Making the article title an `<a>` will match what we're doing for the post cards. (this might need some style changes)